### PR TITLE
#1711 Add new utilities in envDynawo to force the compilation of prea…

### PR DESCRIPTION
**Pull request by gautierbureau**
…ssembled models.

### CODE REVIEW

**Comment by gautierbureau** [comment]
This feature can be used like this:
``` bash
$> ./myEnvDynawo.sh clean-build-models StaticVarCompensator StaticVarCompensatorPV
```

File: util/envDynawo.sh [hunk 1]
```diff
@@ -1049,6 +1052,26 @@ build_tests_coverage() {
   fi
 }
 
+clean_models() {
+  for model in `echo $@`; do
```
Comment (rosiereflo) [inline util/envDynawo.sh:1056]: Is this OK if there is not argument? (just to make sure it would not erase the P folder)

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: util/envDynawo.sh (hunk 1 above)
Comment (gautierbureau) [reply util/envDynawo.sh:1060]: It was ok nothing happened but I added a test to give the information to the user

File: util/envDynawo.sh [hunk 2]
```diff
@@ -1049,6 +1052,33 @@ build_tests_coverage() {
   fi
 }
 
+clean_models() {
+  if [ $# -eq 0 ]; then
+    echo "No model to clean given into argument."
```
Comment (rosiereflo) [inline util/envDynawo.sh:1057]: I would propose "No model given as argument"

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: util/envDynawo.sh (hunk 2 above)
Comment (gautierbureau) [reply util/envDynawo.sh]: Done

**Review by rosiereflo — APPROVED** [review]

**Review by adriguir — APPROVED** [review]

**Pull request merged**